<!-- Make sure launch nav_preapare first -->
<launch>
    <param name="/use_sim_time" value="true" />

    <node type="task5.py" name="task5" pkg="thewalkingdead" output="screen">
    </node>

</launch>